### PR TITLE
Feature/salg 649 survey data model

### PR DESCRIPTION
**Pull request by lats1**

### CODE REVIEW

File: src/Controller/AdminController.php [hunk 1]
```diff
@@ -1,25 +1,166 @@
...
+use EasyCorp\Bundle\EasyAdminBundle\Event\EasyAdminEvents;
 
+/**
+ * Class AdminController.
+ *
+ * Controller for handling requests going to protected area
+ */
 class AdminController extends BaseAdminController
 {
+    /**
```
Comment (tuj) [inline src/Controller/AdminController.php]: Elaborate comments like appendDqlFilterToDqlFilter()

File: src/Controller/AdminController.php [hunk 2]
```diff
@@ -1,25 +1,166 @@
 <?php
 
+/**
+ * @license MIT
+ * @license https://opensource.org/licenses/MIT The MIT License
```
[the remark this answers is not in the document]
Comment (lats1) [reply src/Controller/AdminController.php]: No I wrote it. Codesniffer with Symfony rules enabled insisted on this.

File: src/Repository/SurveyRepository.php [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+     * SurveyRepository constructor.
+     *
+     * @param RegistryInterface $registry
+     */
+    public function __construct(RegistryInterface $registry)
+    {
+        parent::__construct($registry, Survey::class);
+    }
+
+    // /**
```
[the remark this answers is not in the document]
Comment (lats1) [reply src/Repository/SurveyRepository.php]: Resolved by commit dc527c5

File: src/Controller/AdminController.php (hunk 1 above)
Comment (lats1) [reply src/Controller/AdminController.php]: Resolved by commit f4ec6d6

File: src/Controller/AdminController.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (tuj) [reply src/Controller/AdminController.php]: Hmm... I think we should remove it.
Codesniffer should not decide which license we use :)

[the remark this answers is not in the document]
Comment (lats1) [reply src/Controller/AdminController.php]: Allrighty. Resolved by commit 996b8fd

**Pull request merged**